src/coreclr/vm/jitinterface.h

```diff
@@ -523,8 +509,9 @@ class CEEInfo : public ICorJitInfo
                 DestroyHandle(elements[i]);
             }
             delete m_pJitHandles;
-            m_pJitHandles = nullptr;
         }
+
+        delete m_transientDetails;
```

Comment:
SArray has `BITWISE_COPY = TRUE` default argument. It means that it does not call any constructors/destructors for the elements by default. Do we need to explicitly set `BITWISE_COPY = FALSE` so that this calls the element destructors?

Reply:
“There is no curse in Elvish, Entish, or the tongues of Men for this treachery.”

How is this an acceptable design... You are correct, apparently relying on a fundamental C++-ism in a C++ codebase is too much to ask for. Sigh...